app/Http/Controllers/Admin/HomeController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
+{
+    public function index()
```

Comment:
```suggestion
    public function __invoke()
```
Admin/HomeControllerは1つしかアクションを用意しないので__invoke()に変更してください
__invoke()の呼び出しはルート定義でHomeController::classだけ指定で呼び出せるはずです